packages/design-system/src/components/input/stories/index.js

```diff
@@ -34,6 +33,16 @@ import { AlignCenter } from '../../../icons';
 export default {
   title: 'DesignSystem/Components/Input',
   component: Input,
+  args: {
+    label1: 'Normal',
+    label2: 'Suffix',
+    label3: 'Error',
+    label4: 'Disabled',
```

Comment:
💭 Looking back on this I think having the labels as controls isn't really useful. We could probably remove these and keep them static.

Would be especially helpful for the numeric input stories since they have different arguments